clang/test/Driver/sycl-post-link-options-win.cpp

```diff
@@ -3,7 +3,7 @@
 // RUN: %clangxx -### --target=x86_64-pc-windows-msvc -fsycl \
 // RUN:          -Xdevice-post-link -O0 %s 2>&1 \
 // RUN:   | FileCheck -check-prefix OPTIONS_POSTLINK_JIT_OLD %s
```

Comment:
Add `--sysroot=%S/Inputs/SYCL` to the command line as there is no guarantee the base `lib` will be created with the compiler build (for example with the sanitizer based builds).

Reply:
Done.
Thanks very much.